spec.bs

```diff
@@ -2731,10 +2736,18 @@ a {{ReportingBrowserSignals}} |browserSignals|, a [=direct from seller signals=]
      [=generated bid/interest group=]'s [=interest group/ads=] whose
      [=interest group ad/render url=] is |winner|'s
      [=generated bid/ad descriptor=]'s [=ad descriptor/url=].
-  1. If |igAd|'s [=interest group ad/buyer and seller reporting ID=]
-     does not [=map/exist=] and the result of [=query reporting ID k-anonymity count=] given
-     |winner|'s [=generated bid/interest group=] and |igAd| is true:
+  1. If |winner|'s [=generated bid/selected buyer and seller reporting ID=] [=map/exists=]:
+    1. If [=query reporting ID k-anonymity count=] given
+      |winner|'s [=generated bid/interest group=], |igAd|, and |winner|'s [=generated bid/selected buyer and seller reporting ID=] is true:
+      1. If |igAd|'s [=interest group ad/buyer reporting ID=] [=map/exists=],
+        [=map/set=] |reportWinBrowserSignals|["{{ReportWinBrowserSignals/buyerReportingId}}"] to
+        |igAd|'s [=interest group ad/buyer reporting ID=].
+  1. Otherwise if the result of [=query reporting ID k-anonymity count=] given
+    |winner|'s [=generated bid/interest group=], |igAd|, and null is true:
     1. If |igAd|'s [=interest group ad/buyer reporting ID=] [=map/exists=],
+        [=map/set=] |reportWinBrowserSignals|["{{ReportWinBrowserSignals/buyerReportingId}}"] to
+        |igAd|'s [=interest group ad/buyer reporting ID=].
+    1. Otherwise if |igAd|'s [=interest group ad/buyer reporting ID=] [=map/exists=],

```

Comment:
what's the "otherwise"? It seems to be the same condition as the "if", and does the same operation as well?